config/filter.d/lighttpd-auth.conf

```diff
@@ -4,7 +4,13 @@
 [Definition]
 
 failregex = ^: \(http_auth\.c\.\d+\) (password doesn\'t match .* username: .*|digest: auth failed for .*: wrong password|get_password failed), IP: <HOST>\s*$
+            ^: \(\S+\.c\.\d+\) user \S* ?(not found|not authorized|password mismatch|sent invalid token|misbehaved): .+, IP: <HOST>
```

Comment:
`.+` is intended here to capture the additional web-app failure message.
I don't think it will capture any user/client provided data, as this data will not come into the log.
Were you thinking about something specific ?
Thank you ver much @yarikoptic for your feeback 👍